README.md

```diff
@@ -68,3 +68,7 @@ NOTE: If you are an existing CNCF Ambassador and would like to ajdust your locat
 #### What if I want to step down or don't have the time to commit anymore? 
 
 We have an [Emeritus](EMERITUS.md) program to address Ambassadors who'd like to step down from active engagement but still be invited to participate in calls and the community. If you decide to become more active again, emeritus ambassadors are always welcome to be reinstated as full CNCF Ambassadors. Please open up an issue in the repo to start the process.
+
+#### What if I am a student can i apply?
```

Comment:
Campus Ambassador initiative is complimentary, but not a direct part of the career ladder to the CNCF Ambassador program, so I'd remove this paragraph from here.

Reply:
I already deleted that part in the README file